apps/meteor/client/views/room/composer/ComposerUserActionIndicator/ComposerUserActionIndicator.tsx

```
@@ -13,28 +13,29 @@ const ComposerUserActionIndicator = ({ rid, tmid }: { rid: string; tmid?: string
-						users,
-					};
-				})
-				.filter(Boolean) as {
-				action: 'typing' | 'recording' | 'uploading' | 'playing';
-				users: string[];
-			}[],
-		[roomAction],
-	);
+	const actions = useMemo(() => {
+		const entries = Object.entries(roomAction ?? {})
+			.map(([key, _users]) => {
```

Biome (2.4.7)

[error] This callback passed to map() iterable method should always return a value.

(lint/suspicious/useIterableCallbackReturn)